bot/util/utilities.py

```diff
@@ -154,24 +154,145 @@ async def cembed(self, ctx, channel: str, *, json_string: str):
+        a corresponding subcommand. Each of these subcommands takes two arguments (Discord status, activity name),
+        except for `streaming` which takes an additional argument named "stream_url". For removing the currently set
+        activity and reset the bots Discord presence, simply use the `clear` subcommand.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+        """
+        if ctx.invoked_subcommand is None:
+            await ctx.send_help(ctx.command)
+
+    @change_discord_presence.command(name="watching")
+    async def change_discord_presence_watching(self, ctx: commands.Context,
+                                               status: Optional[discord.Status] = discord.Status.online,
+                                               *, activity_name: str):
+        """Command handler for the `presence` subcommand `watching`.
+
+        This is a command that changes the bots Discord presence to a watching activity with the specified name. The
+        Discord status can also be set via the optional status argument.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+            status (Optional[discord.Status]): The status which should be displayed.
+            activity_name (str): The name of the activity shown.
+        """
+        activity = discord.Activity(type=discord.ActivityType.watching, name=activity_name)
+        await self.bot.change_presence(activity=activity, status=status)
+
+    @change_discord_presence.command(name="listening")
+    async def change_discord_presence_listening(self, ctx: commands.Context,
+                                                status: Optional[discord.Status] = discord.Status.online,
+                                                *, activity_name: str):
+        """Command handler for the `presence` subcommand `listening`.
+
+        This is a command that changes the bots Discord presence to a listening activity with the specified name. The
+        Discord status can also be set via the optional status argument.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+            status (Optional[discord.Status]): The status which should be displayed.
+            activity_name (str): The name of the activity shown.
+        """
+        activity = discord.Activity(type=discord.ActivityType.listening, name=activity_name)
+        await self.bot.change_presence(activity=activity, status=status)
+
+    @change_discord_presence.command(name="playing")
+    async def change_discord_presence_playing(self, ctx: commands.Context,
+                                              status: Optional[discord.Status] = discord.Status.online,
+                                              *, activity_name: str):
+        """Command handler for the `presence` subcommand `playing`.
```

Comment:
The docstring does not make clear what values must be passed to make it look like what. especially activity_name is not clear to me. 

Maybe add some Example to the docstring that shows which command would yield which message in discord

Reply:
Slightly adapted the docstrings.